mage_ai/server/app.py

```diff
@@ -90,12 +92,15 @@ def feature_sets():
     ]
     """
     feature_sets = list(map(lambda fs: fs.to_dict(detailed=False), FeatureSet.objects()))
-    valid_feture_sets = [f for f in feature_sets
-                         if set(['column_types', 'statistics']).issubset(f['metadata'].keys())]
+    valid_feture_sets = [
```

Comment:
nit: can we fix the typo in this variable

Reply:
Fixed